### ISSUE TITLE
Image not showing on map

### ISSUE THREAD

**Issue by lanzaiergesi** [nextcloud/maps#713]
The image does not display on the map, and the nextcloud built with php7.4 does not have this problem.
My current version is Nextcloud Hub II (23.0.3)
php version 8.0.17

**Comment by tacruc** [comment]
duplicate #684

**Comment by blackie75** [comment]
It works for me.
My current version of Nextcloud is Nextcloud Hub II (23.0.3)
PHP 8.0.17 

It didn't work before. I gave up and wanted to add photos to the map by hand. To make my work easier, I imported the places I visited from the Maps Me application.Then I added a few photos manually with the location adequate to the data from Maps Me. After some time, I came to the conclusion that I would first import all photos to Nextcloud and then add them manually. After an hour I wanted to start manually adding photos, but to my amazement the photos were placed on the map as it was in previous versions of Nextcloud (according to EXIF ​​data). Finally works!

[Example](https://sakurathai.pl/nextcloud/index.php/s/LrRmZeCDPNoMJkP)
